bindings/nodejs/examples/29-add-voting-event.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * This example creates a new voting event
+ */
+const getUnlockedManager = require('./account-manager');
+
+async function run() {
+    try {
+        const manager = await getUnlockedManager();
+        await manager.getP
```

Comment:
Copy paste went wrong? :trollface:

Reply:
No, this is for another PR :trollface: , I removed it